examples/image/compute_gray_with_squidpy.im.ImageContainer.apply

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+"""
+Convert to grayscale using squidpy.im.ImageContainer.apply
+--------------------
+This example shows how to use :func:`squidpy.ImageContainer.apply` to convert an image layer to grayscale.
+
+We take the mean of all the three channels (RGB) using the defined function `rgb2gray` to convert the image to grayscale.
+The function `rgb2gray` calls numpy method mean to calculate the mean (average) of three channels (RGB) across axis 4.
+
+.. seealso::
+    - :ref:`sphx_glr_auto_examples_image_compute_smooth.py`
+    - :ref:`sphx_glr_auto_examples_image_compute_process_hires.py`
+
+"""
+
+import squidpy as sq
+
+import matplotlib.pyplot as plt
+
+import numpy as np
+
+###############################################################################
+# First, we load the H&E stained tissue image.
+# Here, we only load a cropped dataset to speed things up.
+# In general, :func:`squidpy.im.ImageContainer.apply` can also process very large images
+# (see :ref:`sphx_glr_auto_examples_image_compute_process_hires.py`).
+img = sq.datasets.visium_hne_image_crop()
+
+###############################################################################
+# The method `squidpy.im.ImageContainer.apply` takes a fucntion as an input. The same function or a mapping of {'{library_id}': function} takes a numpy.ndarray as input and produces an image-like output.
```

Comment:
`{'{library_id}': function}` this is not a nice way to specify a function, see how it would render in a notebook.